google-cloud-tasks/samples/test/task_test.rb

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "minitest/autorun"
+require "rack/test"
+require "google/cloud/tasks/v2"
+require_relative "../create_http_task.rb"
+require "cgi"
+
+describe "CloudTasks" do
+  include Rack::Test::Methods
+
+  before do
+    GOOGLE_CLOUD_PROJECT = ENV["GOOGLE_CLOUD_PROJECT"]
```

Comment:
`before` blocks are run before every test. So use instead variables like `@project` instead of constants, so that we can in the future add more tests and not have Ruby complain about redefined constants. An instance variable set in a `before` block will be accessible inside the `it` block.